### ISSUE TITLE
why the loss can not go down?(deeplab)

### ISSUE THREAD

**Issue by YQYI** [tensorflow/models#4144]
I use the pretrain model deeplabv3_mnv2_pascal_train_aug to train supervisely dataset which contains only human and backgrond ,i turn the human label to 15,and background label to 0, just like pascal dataset,but the loss is unstable between 0.1 to 0.7, I think the loss is not good,how can I solve this problem

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by bitfort** [comment]
Thanks for reaching out. I'd recommend stackoverflow as a starting point (tag tensorflow). This github is primarily intended for bug reporting -- if you run into a problem with a bug in code, please feel free to come back and we'll take a look :)

Thanks,
Victor
